ajax-upgradetab.php

```diff
@@ -73,7 +73,8 @@
 global $ajax;
 
 $ajax = true;
-$adminObj = new AdminSelfUpgrade();
+$autoupgradeDir = Tools14::getValue('autoupgradeDir', 'autoupg-inst-' . uniqid());
```

Comment:
can we have autoupgrade-install ? same on the other class ? (it's just wording i know..)